hazelcast/src/test/java/com/hazelcast/test/annotation/LiveTest.java

```diff
@@ -0,0 +1,10 @@
+package com.hazelcast.test.annotation;
+
+
+/**
+ * Annotates tests which requires integration tests over a live api.
+ * <p/>
+ * Will be executed in nightly builds and for jclouds live tests
+ */
+public final class LiveTest {
```

Comment:
I don't know if LiveTest is a good name and including it in sonar is a good option. That makes Sonra Build fragile in every Cloud API down cases. On the other hand, not having this in Sonar Build makes code-coverage report unhappy. There is a trade-off here.